src/app/child-dev-project/children/children.service.ts

```diff
@@ -79,70 +76,46 @@ export class ChildrenService {
         by_child: {
           map: `(doc) => {
             if (!doc._id.startsWith("${ChildSchoolRelation.ENTITY_TYPE}")) return;
-            emit(doc.childId);
-            }`,
+            emit([doc.childId, new Date(doc.start || '3000-01-01').getTime()]);
```

Comment:
why should the fallback start date be in the future rather than 0, considering these as active by default?

Reply:
The idea is that they will be sorted first, compared to the other relations. This has no effect on the `isActive` calculation